HLTrigger/JetMET/interface/HLTPFJetsMatchedToFilteredJetsProducer.h

```diff
@@ -58,7 +58,7 @@ void HLTPFJetsMatchedToFilteredJetsProducer<TriggerJetsRefType>::fillDescription
     edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("src", edm::InputTag("hltPFJets"));
-  desc.add<edm::InputTag>("triggerJetsFilter", edm::InputTag("hltSingleJet240Regional"));
+  desc.add<edm::InputTag>("triggerJetsFilter", edm::InputTag("hltCaloJetsFiltered"));
```

Comment:
This default value is unimportant for HLT's purposes (always overriden in the configuration). `hltCaloJetsFiltered` just seemed less confusing/arbitrary than `hltSingleJet240Regional`.